packages/quantic/force-app/main/default/lwc/quanticHeading/quanticHeading.js

```diff
@@ -0,0 +1,31 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticHeading` component offers the ability to display a label with a customizable heading level.
+ * @category Utility
+ * @example
+ * <c-quantic-heading label="My label" level="1"></c-quantic-heading>
+ */
+export default class QuanticHeading extends LightningElement {
+  /**
+   * The label dsiplayed inside the heading.
+   * @api
+   * @type {string}
+   */
+  @api label;
+  /**
+   * The level to use for the heading label, accepted values are integers from 1 to 6.
+   * A value outside of the range of 1 to 6 will render a div instead of a heading.
+   * @type {number}
+   */
+  @api level;
+
+  renderedCallback() {
+    const heading = this.template.querySelector('[data-role="label"]');
+    // @ts-ignore
+    const level = parseInt(this.level, 10);
+    const headingTag = level > 0 && level <= 6 ? `h${level}` : 'div';
+    // eslint-disable-next-line @lwc/lwc/no-inner-html
```

Comment:
we can disable it at file level but I don't really see the value of disabling it since it's used just once.

Reply:
I said folder :) 

I'm assuming we have that in multiple places in quantic /lwc folder, perhaps we could remove that rule anyway across all the project since it's a lwc rule?

Reply:
actually no need need to disable this rule anymore, I'm no longer using innerHTML, replaced it with the `appendChild` method.